pkg/kubernetes/scripts/virtual-machines/kube-middleware.jsx

```diff
@@ -49,4 +51,10 @@ async function vmDelete({ vm }) {
     return kubeMethods.delete(selfLink); // no value is returned (empty promise). An exception is thrown in case of failure
 }
 
-export { vmDelete, vmCreate };
+async function fetchStatSummary(nodeName) {
+    const path = "/api/v1/nodes/" + encodeURIComponent(nodeName) + "/proxy/stats/summary/";
```

Comment:
I guess you already did research on that, but just to be sure: Can't this be continuous feed of data (like `watch`)?

If so, we can just turn it on-off per node, if needed.
That would be probably the most effective way.

Reply:
> I guess you already did research on that, but just to be sure: Can't this be continuous feed of data (like watch)?

It could be done, but we would have to enhance kube-client code to support this (not so simple as adding another entity). 

> If so, we can just turn it on-off per node, if needed.

That would need to be implemented there as well

#### Summary

Already implemented solution is quite simple and works ok. We can implement this in kube-client if we want more abstract and general version (which would take some time). Kube-client also uses timeout which is probably too often, so we would have to also parametrize that.